### PR TITLE
Feature/5289 qa cert data UI

### PR DESCRIPTION
**Pull request by Qb10**

### CODE REVIEW

File: src/components/SystemAdmin/SubmissionAccess/FilterFormAdmin/FilterFormAdmin.js [hunk 1]
```diff
@@ -331,10 +335,14 @@ const FilterFormAdmin = ({
                   : " "
               }
             >
+<<<<<<< HEAD
```
Comment (fanwari1) [inline src/components/SystemAdmin/SubmissionAccess/FilterFormAdmin/FilterFormAdmin.js]: looks like a merge conflict here

File: src/components/SystemAdmin/SubmissionAccess/FilterFormAdmin/FilterFormAdmin.js [hunk 2]
```diff
@@ -331,10 +335,14 @@ const FilterFormAdmin = ({
...
+              <Button onClick={clearFilters} outline={true}>
+=======
               <Button
                 onClick={clearFilters}
                 outline={true}
               >
+>>>>>>> 7df9491faa092e703bab8c2a8bd2b853e46b598e
```
Comment (subrataroyCVP) [inline src/components/SystemAdmin/SubmissionAccess/FilterFormAdmin/FilterFormAdmin.js]: Conflicts

**Review by fanwari1 — APPROVED** [review]
lgtm

**Review by subrataroyCVP — APPROVED** [review]
LGTM

**Pull request merged**